catalogue/webapp/app.js

```diff
@@ -1,3 +1,11 @@
+// This needs to be the first module loaded in the application
+if (process.env.NODE_ENV !== 'test') {
+  require('elastic-apm-node').start({
+    serviceName: process.env.NEXT_PUBLIC_APM_SERVICE_NAME,
+    ...require('@weco/common/services/apm/apmConfig'),
+  });
+}
+
```

Comment:
Just a thought, how might we do this if we remove this entrypoint and use standard nextjs?

Reply:
I don't think we'd be able to do that tbh - think we'd need the custom server for this alone

Reply:
Why I am asking:

![Screenshot 2021-09-24 at 11 44 42](https://user-images.githubusercontent.com/31692/134662324-c8edbf69-24a4-4a70-82ff-b208f058c9da.png)

-- [from here](https://nextjs.org/docs/advanced-features/custom-server)